sys/cddl/dev/dtrace/mips/dtrace_isa.c

```diff
@@ -574,7 +577,17 @@ dtrace_next_uframe(register_t *pc, register_t *sp, register_t *ra)
 			registers_on_stack = 1;
 			break;
 		}
+
+		/* The last userspace function called before a syscall starts
```

Comment:
It's not really a function, it's a trampoline (which is needed for lazy binding) that jumps to the real function.

Does this ustack() also work if you run the userspace program with `export LD_CHERI_BIND_NOW=1`?

Reply:
What should change with `LD_CHERI_BIND_NOW`? unfortunately, the real `ustack` tests depend on Perl and ksh, so it is difficult to say if it really works. (btw, they were only 3 tests) Can you imagie a testcase?